include/tables/ProjectTable.h

```diff
@@ -116,15 +117,32 @@ class ProjectTable : public DBTable<ProjectRow> {
     addColumn("projectname");
     addColumn("username");
     addColumn("description");
-    ProjectCache::getInstance(lru_cap, "Project");
+    ProjectCacheById::getInstance(lru_cap, "ProjectCacheById");
+    ProjectCacheByName::getInstance(lru_cap, "ProjectCacheByName");
   }
 
   ProjectRow get(Ndb* connection, int projectId) {
     ProjectRow row = doRead(connection, projectId);
-    ProjectCache::getInstance().put(row.mId, row.mProjectName);
     return row;
   }
 
+  ProjectRow getByName(Ndb* connection, std::string projectName) {
+    AnyMap args;
+    args[1] = projectName;
+    ProjectVec projects = doRead(connection, getColumn(1), args);
```

Comment:
is the index name `projectname` ?

Reply:
updated the code to use the name directly. I assume this creates and index with projectname as name... right?
https://github.com/logicalclocks/hopsworks-chef/blob/master/files/default/sql/ddl/3.3.0__initial_tables.sql#L782